schemas/rule.yml

```diff
@@ -81,4 +81,31 @@
 
       example: https://en.wikipedia.org/wiki/DNS_over_TLS
 
+    - name: author
+      level: extended
+      type: keyword
+      short: Rule author
+      description: >
+        Name, organization, or pseudonym of the author or authors who created the rule used to generate this event.
+
+      example: John R. Doe, Elastic, s3cst1ltsk1n
+      normalize:
+        - array
+
+    - name: license_type
+      level: extended
+      type: keyword
+      short: Rule license
+      description: >
+        Name of the license under which the rule used to generate this event is made available.
+
+      example: Apache 2.0
+
+    - name: license_reference
```

Comment:
I'm not sure the reference URL of the license of the rule should be added to every single alert. That would take a whole lot of room for very little benefit, no?

Reply:
Good question.  Yes, this does seem like very little benefit to most events/alerts that would be received.  It would be most valuable to a rule document itself, which is not technically an ECS event.

Reply:
I've removed this field altogether for now.  If we build a strong case for it later, we can add with another PR.